src/Components/AllTrips.js

```diff
@@ -0,0 +1,162 @@
+      });
+    }
+  };
+
+  useEffect(() => {
+    if (searchValue.citySearch && searchValue.countrySearch) {
+      const cityData = trips.filter((trip) => {
+        return trip.tripInfo.city.includes(searchValue.citySearch);
+      });
+
+      const newData = cityData.filter((trip) => {
+        return trip.tripInfo.country.includes(searchValue.countrySearch);
+      });
+
+      setFilteredTrips(newData);
+    } else if (searchValue.citySearch && !searchValue.countrySearch) {
+      const newData = trips.filter((trip) => {
+        return trip.tripInfo.city.includes(searchValue.citySearch);
+      });
+
+      setFilteredTrips(newData);
+    } else if (!searchValue.citySearch && searchValue.countrySearch) {
+      const newData = trips.filter((trip) => {
+        return trip.tripInfo.country.includes(searchValue.countrySearch);
+      });
+
+      setFilteredTrips(newData);
+    } else {
+      setFilteredTrips([]);
+    }
+  }, [searchValue, trips]);
+
+  return (
+    <AnimatedPage>
+      <section className="all-trips">
+        <form>
+          <div className="input-container">
+            <label htmlFor="countrySearch">Country:</label>
+            <select id="countrySearch" onChange={handleSearchChange}>
+              <option value="">Select:</option>
+              {countryData.length
+                ? countryData.map((container, indexNumber) => {
+                    return (
+                      <option
+                        key={indexNumber + container.country}
+                        value={container.country}
+                      >
+                        {container.country}
+                      </option>
+                    );
+                  })
+                : undefined}
+            </select>
+          </div>
+          <div className="input-container">
+            <label htmlFor="citySearch">City:</label>
+            <select id="citySearch" onChange={handleSearchChange}>
+              <option value="">Select:</option>
+              {countryData.length
+                ? countryData.map((container) => {
+                    if (container.country === searchValue.countrySearch) {
+                      return container.cities.map((city) => {
+                        return (
+                          <option
+                            key={city + searchValue.countrySearch}
+                            value={city}
+                          >
+                            {city}
+                          </option>
+                        );
+                      });
+                    }
+                  })
+                : undefined}
+            </select>
+          </div>
+        </form>
+        <div className="trips">
+          {(filteredTrips.length ? filteredTrips : trips)
+            .slice(0)
+            .reverse()
+            .map((trip) => {
+              return (
+                <>
+                  <div key={trip.key} className="trip-container">
```

Comment:
kebab case on className but I've also seen you use Camel Case *i.e. safetyRating

Reply:
good 👁! switched all of my camel rebels to kebab!